Source/automap.cpp

```diff
@@ -470,6 +470,9 @@ AutomapTile GetAutomapTypeView(Point map)
 		}
 		return { AutomapTile::Types::None, AutomapTile::Flags::Dirt };
 	}
+	// This tile incorrectly has flags for both HorizontalArch and VerticalArch in the amp data

```

Comment:
Patch the AMP data on load rather then have a special run time case for it. Better to have 1 byte set per level load then two comparison, a multiplication and an addition every frame.